src/box/box.cc

```diff
@@ -1535,6 +1535,15 @@ box_init_say()
 	return 0;
 }
 
+int
+box_say_log_level()
+{
+	if (luaT_dostring(tarantool_L,
+			  "require('log').box_api.say_log_level()") != 0)
```

Comment:
Donno. I would just use something like `"require('log').info(require('log').cfg.level)"`. To me introducing the function only for a log line is too much.

Reply:
Removed this code.